llarp/router/i_outbound_message_handler.hpp

```diff
@@ -24,9 +24,9 @@ namespace llarp
 
   using SendStatusHandler = std::function<void(SendStatus)>;
 
-  static const size_t MAX_PATH_QUEUE_SIZE = 50;
-  static const size_t MAX_OUTBOUND_QUEUE_SIZE = 200;
-  static const size_t MAX_OUTBOUND_MESSAGES_PER_TICK = 100;
+  static const size_t MAX_PATH_QUEUE_SIZE = 100;
+  static const size_t MAX_OUTBOUND_QUEUE_SIZE = 1000;
+  static const size_t MAX_OUTBOUND_MESSAGES_PER_TICK = 500;
 
```

Comment:
If these queue sizes are really improving things, then we should probably take a closer look at them. An easy start would be to print out any time we hit a new ceiling on size, then make sure we aren't filling the queue.

Also, I'd be interested to see if there's a direct correlation between queue size and bandwidth being pushed. Assuming this is the case, we don't want to unconditionally tune our queue size for someone with a large pipe (1Gbit, let's say) as  this would be inappropriate for someone with a small pipe (5Mbit, for example).